core/model/user-interface-descriptors/ntp-server-user-interface-descriptor.mjson

```diff
@@ -0,0 +1,19 @@
+{
+    "root": {
+        "prototype": "montage/core/meta/user-interface-descriptor",
+        "properties": {
+            inspectorComponentModule: {
```

Comment:
This has to be a valid JSON file, you need to put `"` around the properties names